rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -195,6 +196,7 @@ def create_inquirer(
         data_dir=data_directory,
         cryptocompare=cryptocompare,
         coingecko=gecko,
+        manualcurrent=ManualCurrentOracle(),
```

Comment:
It feel strange the way things are initialized here. Maybe can you just move all the initializations to the arguments?

Reply:
Ok